runtime-modules/content/src/permissions/curator_group.rs

```diff
@@ -13,6 +57,9 @@ where
 
     /// When `false`, curator in a given group is forbidden to act
     active: bool,
+
+    // Group's moderation permissions
+    permissions: ModerationPermissionsByLevel<T>,
```

Comment:
`permissions` => `level_permissions` or `permissions_by_level` ? what do you think

Reply:
Renamed to `permissions_by_level`